backend/src/projekti/adapter/adaptToAPI/adaptVuorovaikutusKierros.ts

```diff
@@ -151,14 +192,42 @@ function adaptVuorovaikutusTilaisuusJulkaisut(
     const vuorovaikutusTilaisuudetCopy = cloneDeep(vuorovaikutusTilaisuudet);
     return vuorovaikutusTilaisuudetCopy.map((vuorovaikutusTilaisuus) => {
       const yhteystiedot: Yhteystieto[] | undefined = vuorovaikutusTilaisuus.yhteystiedot;
+      const nimi: LocalizedMap<string> | undefined = vuorovaikutusTilaisuus.nimi;
+      const Saapumisohjeet: LocalizedMap<string> | undefined = vuorovaikutusTilaisuus.Saapumisohjeet;
+      const osoite: LocalizedMap<string> | undefined = vuorovaikutusTilaisuus.osoite;
+      const paikka: LocalizedMap<string> | undefined = vuorovaikutusTilaisuus.paikka;
+      const postitoimipaikka: LocalizedMap<string> | undefined = vuorovaikutusTilaisuus.postitoimipaikka;
       delete vuorovaikutusTilaisuus.yhteystiedot;
+      delete vuorovaikutusTilaisuus.nimi;
+      delete vuorovaikutusTilaisuus.Saapumisohjeet;
+      delete vuorovaikutusTilaisuus.osoite;
+      delete vuorovaikutusTilaisuus.paikka;
+      delete vuorovaikutusTilaisuus.postitoimipaikka;
       const tilaisuus: API.VuorovaikutusTilaisuusJulkaisu = {
-        ...(vuorovaikutusTilaisuus as Omit<API.VuorovaikutusTilaisuusJulkaisu, "yhteystiedot">),
+        ...(vuorovaikutusTilaisuus as Omit<
+          API.VuorovaikutusTilaisuusJulkaisu,
+          "yhteystiedot" | "nimi" | "Saapumisohjeet" | "osoite" | "paikka" | "postitoimipaikka"
+        >),
```

Comment:
Sama homma täällä

Reply:
Näitä oli useammassa paikassa tääl bäkkärissä ja suosittelisin vahvasti muokkaamaan nää sellasiks, ettei deletoida kenttiä vaan valitaan ne kentät, jotka halutaan palauttaa.